meson.build

```diff
@@ -0,0 +1,60 @@
+project('rhino-setup',
+        'rust',
+          version: '0.1.0',
+    meson_version: '>= 0.59.0',
+    license: 'GPL-3.0-or-later'
+)
+
+i18n = import('i18n')
+gnome = import('gnome')
+
+base_id = 'org.rhinolinux.RhinoSetup'
+
+dependency('glib-2.0', version: '>= 2.66')
+dependency('gio-2.0', version: '>= 2.66')
+dependency('gtk4', version: '>= 4.0.0')
+dependency('libadwaita-1', version: '>= 1.0.0')
+
+glib_compile_resources = find_program('glib-compile-resources', required: true)
```

Comment:
I think this is never used? Meson will internally use glib-compile-resources, and handles checking for it on its own.